owners/index.js

```diff
@@ -34,85 +36,55 @@ module.exports = app => {
     level: process.env.LOG_LEVEL || 'info',
   });
 
-  /**
-   * Runs the steps to create or update an owners-bot check-run on a GitHub Pull
-   * Request.
-   *
-   * @param {!GitHub} github GitHub API interface.
-   * @param {!PullRequest} pr pull request to run owners check on.
-   */
-  async function runOwnersCheck(github, pr) {
-    const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
-    const ownersCheck = new OwnersCheck(localRepo, github, pr);
-    let checkRunId;
-    let latestCheckRun;
+  /** Health check server endpoints **/
+  // TODO(rcebulko): Implement GitHub authentication to prevent spamming any of
+  // these endpoints.
+  router.get('/status', (req, res) => {
+    res.send('The OWNERS bot is live and running!');
```

Comment:
Does AppEngine expose version numbers in process.env? If so you can conditionally add it here

Reply:
Done.